holoviews/tests/core/test_dynamic.py

```diff
@@ -740,7 +740,7 @@ def fn2(x, y):
 
         overlaid = dmap * dmap2
         overlay = overlaid[()]
-        self.assertEqual(overlay.Scatter.I, fn(0, 0))
+        self.assertEqual(overlay.Scatter.I, fn(None, None))
```

Comment:
At some point, it could be that the dataframe in scatter returned (0, 0). But now it returns (None, None)